### PR TITLE
Final blogpost - 2018-09-17-sectumsempra-rgosc.md

### PR DESCRIPTION
**Pull request by avneetreen**

### CODE REVIEW

File: blog/_posts/2018-09-17-sectumsempra-rgsoc.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+---
+title: “How lucky are we to have something that makes saying goodbye so hard!”
+layout: post
+created_at: Mon Sep 17 2018
+permalink: blog/2018-09-17-team-sectumsempra
```
Comment (v-thakkar) [inline blog/_posts/2018-09-17-sectumsempra-rgsoc.md]: Hi @avneet14027 ! Thanks for the PR. Amazing blog post and quiet very well written with the mention of personal experiences. 

Minor comment here. Permalink and and file name should be the same. Please fix that.

File: blog/_posts/2018-09-17-sectumsempra-rgsoc.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+<div class="image-credits">Source: Giphy</div>
+
+### Next Steps
+We feel that we would now be able to make a smooth transition into the world of open source. We are no more dubious of our selves while trying out to figure out how to contribute to open source and whether what we do would be suffice,  we would now definitely just dive right in! 
+And we feel there are no better words to express how we feel for RGSoC than to say.
+> *“There are no goodbyes for us. Wherever you are, you will always be in my heart” ~ Mahatma Gandhi*
+
+Alvida, RGSoC!
+
+![Alt Text](https://github.com/avneet14027/images/blob/master/8cf94510-9dba-0133-a027-0e7c926a42af.gif)
```
Comment (v-thakkar) [inline blog/_posts/2018-09-17-sectumsempra-rgsoc.md]: You seem to have forgot to add image credits here. :)

**Review by v-thakkar — CHANGES_REQUESTED** [review]

File: blog/_posts/2018-09-17-sectumsempra-rgsoc.md (hunk 1 above)
Comment (avneetreen) [reply blog/_posts/2018-09-17-sectumsempra-rgsoc.md]: Thanks @nerdyvaishali ! We enjoyed writing the blog post. Straight from the heart ❤️ 
I fixed the file name and  added the image credits.

Regards,
Avneet

**Review by v-thakkar — APPROVED** [review]

File: blog/_posts/2018-09-17-team-sectumsempra.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+asking for favour or help, but ASK! Don’t shy away.
+
+- **Set the bars high for yourselves**: In our experience, we have observed that when it
+comes to career, expectation bar is embarrassingly low for women. And this is majorly
+because why we don’t see ourselves being able to match men. Think that you’ll achieve
+big, and so you will!
+
+Kate Winslet just puts it right:
+
+[![Kate Winslet advice to women](http://img.youtube.com/vi/DeRcIf4sZiM/0.jpg)](http://www.youtube.com/watch?v=DeRcIf4sZiM "Kate Winslet advice to women")
```
Comment (v-thakkar) [inline blog/_posts/2018-09-17-team-sectumsempra.md]: Hi @avneet14027 ! You added bunch of more images/GIFs in the file and seem to have forgot to add image credits. Please add them here.

File: blog/_posts/2018-09-17-team-sectumsempra.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+help you.” And, this is how we grow. TOGETHER.
+
+![Alt Text](https://github.com/avneet14027/images/blob/master/helping-others.gif)
+<div class="image-credits">Source: Giphy</div>
+
+- **YAS Girl, you can do it!** The feeling that “I’m not ready. Not just yet” always clouds us
+too often. You’ll never be prepared, there’s always be a scope for improvement. But,
+don’t let that hold you back. If you so want something, go get it. Period.
+
+![Alt Text](https://themotivationmentalist.files.wordpress.com/2014/05/dont-ever-let-someone-tell-you-that-you-cant-do-something.jpg)
```
Comment (v-thakkar) [inline blog/_posts/2018-09-17-team-sectumsempra.md]: Image credits missing.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

File: blog/_posts/2018-09-17-team-sectumsempra.md (hunk 3 above)
Comment (avneetreen) [reply blog/_posts/2018-09-17-team-sectumsempra.md]: Hi @nerdyvaishali , I have added the image credits now.
Thanks,
Avneet

**Comment by Rupal-IIITD** [comment]
Hi, @nerdyvaishali 

We made the required changes as suggested by you. Kindly feel free to let us know if there are any more steps remaining before the PR is merged. :)

**Review by v-thakkar — APPROVED** [review]

**Comment by jnoni** [comment]
@nerdyvaishali @anasofiapinho 
When are we going to publish our blog? Too excited to see it up. :dancer:

**Comment by v-thakkar** [comment]
@Rupal-IIITD I was sick last week so we couldn't publish many blog posts. We'll start publishing blog posts this week again.

**Pull request merged**